addons/web/static/src/js/view_form.js

```diff
@@ -4379,6 +4379,10 @@ instance.web.form.One2ManyViewManager = instance.web.ViewManager.extend({
 instance.web.form.One2ManyDataSet = instance.web.BufferedDataSet.extend({
     get_context: function() {
         this.context = this.o2m.build_context();
+        var self = this;
+        _.each(arguments, function(context) {
```

Comment:
you should add an argument context like in the [get_context of DataSet](https://github.com/odoo/odoo/blob/780dd9891f5783fd07a075a445314df912d0a88c/addons/web/static/src/js/data.js#L674)